.github/workflows/agent-validation.yml

```
@@ -0,0 +1,86 @@
+          BASE_REF: ${{ env.BASE_REF }}
+          HEAD_REF: ${{ env.HEAD_REF }}
+        run: |
+          set -euo pipefail
+          echo "PR_TITLE: $PR_TITLE"
+          echo "BASE_REF:  $BASE_REF"
+          echo "HEAD_REF:  $HEAD_REF"
+          
+          # Compute changed files against base
+          CHANGED=$(git diff --name-only "$BASE_REF...HEAD" || true)
+          echo "Changed files:\n$CHANGED"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)